src/ecmech/ECMech_evptn.h

```diff
@@ -658,8 +658,11 @@ namespace ecmech {
                //// need shrate%eff instead
                //// CALL calc_pl_eff(dp_def_rate_contrib, pl_vecd, detV%ri)
                // CALL setup_ss_shrate_vals(shrate_l, crys%tmp1_slp, zero, .TRUE.)
+#if defined(ECMECH_USE_DPEFF)
+               _shrate_eff_contrib = vecd_Deff(pl_vecd);
```

Comment:
It occurred to me that if you make this change here that you may also want to change the name of the history variables from "shrate_eff" and "shr_eff" to something else like "dplas_eff" and "eps" ... with those changes being in ECMech_evptnWrap.h

Reply:
Good catch. I added a new ifdef in evptnWrap that assigns the appropriate name if this ifdef is defined.